homeassistant/components/switch/rpi_servo.py

```diff
@@ -0,0 +1,119 @@
+"""
+Allows to configure a switch using RPi GPIO Servos.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.rpi_gpio/
+"""
+import logging
+
+import voluptuous as vol
+import homeassistant.components.rpi_gpio as GPIO
```

Comment:
Don't use capital letters for the imported component. It's a module not a constant.

Reply:
Okay, thanks for the review :+1:. I'm not entirely familiar with the python naming conventions (or pyhton at all for that matter). I will add these changes as soon as I can find the time for it.